test/unit/model/define.test.js

```diff
@@ -117,7 +117,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           }
         });
 
-        if (dialect === 'postgres' || dialect === 'sqlite' || dialect === 'mssql') {
+        if (['postgres', 'mariadb', 'mssql'].includes(dialect)) {
```

Comment:
should be pg, sqlite, mssql

Reply:
since no test failed, I think we are lacking some

Reply:
Yup, will add some

Reply:
Did you add a test?

Reply:
> Yup, will add some

Actually ... it was a test already?!